client/my-sites/checkout/src/components/test/secondary-cart-promotions.tsx

```diff
@@ -20,8 +23,51 @@ jest.mock( '@automattic/calypso-config', () => {
 	return mock;
 } );
 
+jest.mock( '@automattic/data-stores', () => {
+	return {
+		...jest.requireActual( '@automattic/data-stores' ),
+		Plans: {
+			...jest.requireActual( '@automattic/data-stores' ).Plans,
+			usePlans: jest.fn(),
+			useSitePlans: jest.fn(),
+		},
+	};
+} );
+
 describe( 'SecondaryCartPromotions', () => {
 	const store = applyMiddleware( thunk )( createStore )( storeData );
+	const queryClient = new QueryClient();
+	const Wrapper = ( { children } ) => (
+		<QueryClientProvider client={ queryClient }>{ children }</QueryClientProvider>
+	);
+
+	beforeEach( () => {
+		jest.clearAllMocks();
+		Plans.usePlans.mockImplementation( () => ( {
+			isLoading: false,
+			data: {},
+		} ) );
+		Plans.useSitePlans.mockImplementation( () => ( {
+			isLoading: false,
+			data: {
+				[ PLAN_PERSONAL ]: {
+					expiry: null,
+					pricing: {
+						billPeriod: 365,
+						currencyCode: 'USD',
+						originalPrice: {
+							full: 500,
+							monthly: 500,
+						},
+						discountedPrice: {
+							full: 250,
+							monthly: 250,
+						},
+					},
+				},
+			},
+		} ) );
```

Comment:
@sirbrillig you may not like the way I mock the data here. We have a couple of options I think - we could import the mocks from `/packages/data-stores/src/plans/mocks/next`, or mock the pricing hook `usePricingMetaForGridPlans`, or mock the underlying endpoints? Not sure what's preferred. WDYT?

Reply:
https://github.com/Automattic/wp-calypso/pull/89472#discussion_r1562357889